src/app.js

```diff
@@ -131,16 +131,18 @@ class Application {
                     'pt', 'pth', 't7',
                     'pkl', 'joblib',
                     'pbtxt', 'prototxt',
+                    'armnn',
                     'cfg',
                     'xml' ] }
             ]
         };
-        const selectedFiles = electron.dialog.showOpenDialogSync(showOpenDialogOptions);
```

Comment:
Looks like this failed to merge?

Reply:
sorry for confusing, this is mis-pushed one.

commit f6a3411 makes problem with opening file in my Mac OS X, I revert the commit for testing. and it should not be pushed.